proof/refine/RISCV64/StateRelation.thy

```diff
@@ -188,6 +318,22 @@ primrec aobj_relation_cuts :: "RISCV64_A.arch_kernel_obj \<Rightarrow> machine_w
 | "aobj_relation_cuts (PageTable pt) x =
      (\<lambda>y. (x + (ucast y << pteBits), pte_relation y)) ` UNIV"
 
+abbreviation
+  sc_relation_cut :: "Structures_A.kernel_object \<Rightarrow> Structures_H.kernel_object \<Rightarrow> bool"
+  where
+  "sc_relation_cut obj obj' \<equiv>
```

Comment:
style: name should be on same line as `abbreviation`, `where` on same line as type